### DISCUSSION TITLE
2023/06/05 Task

### DISCUSSION THREAD

**Discussion by JeongJaeSoon** [2P3S/scrum-dice#58] [General]
# 승현
* Front End refactoring
* API 연동 준비

# 재순
* scurm-dice 에 필요한 상세 기능 구현

# 승형
* AWS 서버에 scrum-dice 배포

**Comment by hyunnnn98** [comment]
# 😄 프론트 리펙토링
- [x] 아토믹 디자인 기반으로 컴포넌트 분리
- [x] 데이터마다 스토어 분리
- [ ] 다국어 지원

**Comment by JeongJaeSoon** [comment]
# scurm-dice 에 필요한 상세 기능 구현
* [x] REST API, Socket 데이터 정리
* [x] Socket 단순화 
* [x] scrume-dice 카드 entity 정리
* [x] logic 에 필요한 사전 구현 실시
